lib/gpm.js

```diff
@@ -23,7 +23,7 @@ const path = require('path');
 const semver = require('../vendor/semver');
 const readFile = util.promisify(fs.readFile);
 const access = util.promisify(fs.access);
-const {execFile} = child_process;
+const {execFile, spawn} = child_process;
```

Comment:
If `execFile` is not used anymore, we should remove it from this import statement.